applications_tests/gls_vans_3d/packed_bed.prm

```diff
@@ -0,0 +1,146 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set number mesh adapt       = 0
+  set output name              = result_
+  set output frequency	  	= 1
```

Comment:
I think we are not supposed to write output files in application tests. It's better to increase the output frequency, so that the test doesn't write any output files.